dependent-mapping/README.md

```diff
@@ -0,0 +1,30 @@
+---  
+layout: pattern  
+title: Dependent Mapping
+folder: dependent-mapping
+permalink: /patterns/dependent-mapping/  
+categories: Behavioral
+tags:
+- Database
```

Comment:
The available tags are listed on page https://github.com/iluwatar/java-design-patterns/wiki/07.-Categories-and-Tags
I recommend using `Data access` here.

Reply:
Please specify the language as well.

Reply:
Excuse me, I want to know what "language" means? :)